src/dbos.ts

```diff
@@ -552,8 +577,8 @@ export class DBOS {
   //////
   static globalLogger?: DLogger;
```

Comment:
globalLogger should also be private. We have a `static get logger()` function.

Reply:
Actually, we might not want to expose any globals. Instead, we can provide getters for them.

Reply:
Actually, it doesn't look like we read the `DBOS.globalLogger` field ever. We set it in a couple places and we have one test that checks to see that it's defined. But the `DBOS.logger` getter uses the one from context if available, the one from executor if available or constructs a new GlobalLogger.